mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -1044,17 +1045,9 @@ func containerNodePoolTaintSuppress(k, old, new string, d *schema.ResourceData)
 
 	root := k[:idx]
 
-	// Right now, GKE only applies its own out-of-band labels when you enable
-	// Sandbox. We only need to perform diff suppression in this case;
-	// otherwise, the default Terraform behavior is fine.
-	o, n := d.GetChange(root + ".sandbox_config.0.sandbox_type")
```

Comment:
Did GKE change its behavior here? Now it always applies the diff suppression?

I think this is probably safe to change, but just want to understand why it needs to.